test/arrayops.jl

```diff
@@ -381,6 +381,14 @@ end
 
     @test findall(in([1, 2]), 2) == [1]
     @test findall(in([1, 2]), 3) == []
+
+    @test sort(findall(Dict(1=>false, 2=>true, 3=>true))) == [2, 3]
+
+    @test findall(true) == [1]
+    @test findall(false) == Vector{Int}[]
```

Comment:
I guess you meant
```suggestion
    @test findall(false) == Int[]
```

Kinda weird that
```
julia> Int[] == Vector{Int}[]
true
```
but I guess all elements are equal so...